### DISCUSSION TITLE
Run gamd-openmm in parallel

### DISCUSSION THREAD

**Discussion by cpayal** [MiaoLab20/gamd-openmm#26] [Q&A] [answered]
Hi 

Is there a way to run gamd-openmm in parallel ? I am using a linux workstation and would like to try out gamd on my system using CPUs  .

Thanks

**Comment by lvotapka** [comment, marked as answer]
If you run the gamdRunner program with the '-h' argument, you should see instructions for how to run in the CPU platform. By default, OpenMM runs on all available cores unless the OPENMM_CPU_THREADS environmental variable says otherwise.

**Reply by cpayal** [reply]
Thanks @lvotapka 

I see ! yes, I was already using "CPU" as the platform with -p flag but was wondering if the exact number could be specified, to adjust CPU utilization with other running jobs. 

Thanks
